### ISSUE TITLE
forceUpdate

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#511]
Originally reported on Google Code with ID 240

```
Hi,
my propose is to introduce a "forceUpdateOnChange" in order to refetch
events at every change: viewChange and render.

Attach is fullcalendar.js modified, look at "BC" for changes.


```

Reported by `nicola.biada` on 2009-12-11 07:41:39

<hr>
- _Attachment: [fullcalendar.js](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-240/comment-0/fullcalendar.js)_

**Comment by hisuwh** [comment]
What happened to this? its hard to follow?
